pkg/handler/handler.go

```diff
@@ -183,9 +184,23 @@ func (h handler) bind(w http.ResponseWriter, r *http.Request, params map[string]
 		return
 	}
 
+	// process binding request
 	resp, err := h.broker.Bind(instanceUUID, bindingUUID, req)
 
-	writeDefaultResponse(w, http.StatusCreated, resp, err)
+	if err != nil {
+		switch err {
+		case broker.ErrorDuplicate:
+			writeResponse(w, http.StatusConflict, broker.BindResponse{})
+		case broker.ErrorAlreadyProvisioned:
```

Comment:
Nit pick: Is this really an error if we're returning ```http.StatusOK```? You don't have to change this now, but maybe the variable should be renamed.